components/README.md

```diff
@@ -0,0 +1,23 @@
+# Components
+
+## Overview
+
+The components folder contains the sources of all Kyma components.
+A Kyma component is any Pod/container/image deployed with and referenced in a Kyma module/chart to provide the module's functionality.
+Each subfolder in the _components_ directory defines one component.
+
+## Details
+
+Every Kyma component has a dedicated folder containing its sources and a README.md containing further instructions on how to build and develop the component.
+
+The component name and with that the folder name should be suffixed with its component type. The types are defined as following:
```

Comment:
I vote for 2. option. However it is not only the folder name, it will be also the component name. So whenever we talk about the component later, we will use the folder name.